src/components/Common/CaseStudyCards/CaseStudyHero.js

```diff
@@ -28,10 +28,10 @@ const SpecialitiesAndServices = ({ specialities, services }) => (
   </Padding>
 )
 
-const RightHandText = ({ title, specialities, services }) => (
+const RightHandText = ({ title, specialities, services, as = 'h2' }) => (
   <Fragment>
     <Padding bottom={0.5}>
-      <SectionTitle>{title}</SectionTitle>
+      <SectionTitle as={as}>{title}</SectionTitle>
```

Comment:
why do we need to define as H2 here, if under the hood it's 
```
const SectionTitleSectionTitle = styled.h2`
  ${SectionTitleStyles}`
```
Is it because when we make Styled components it overrides the H2 aspect?

Reply:
yep, you can pass `as` as a prop to a styled component and it will change the rendered HTML tag